packages/botonic-nlp/src/preprocess/codifier.ts

```diff
@@ -1,6 +1,9 @@
 export class Codifier {
-  constructor(public vocabulary: string[], readonly categorical: boolean) {
+  categorical: boolean
+
+  constructor(public vocabulary: string[], args = { categorical: false }) {
```

Comment:
normally, it's better to add a verb before a property: `categorical` --> `isCategorical`.
When using this variable you will see it more logical:
```
if(this.categorical) --> weird
if(this.isCategorical) --> makes more sense to me
```